mkosi/sandbox.py

```diff
@@ -129,7 +129,7 @@ class Style:
 
 
 def oserror(syscall: str, filename: str = "") -> None:
-    if ctypes.get_errno() == ENOSYS:
+    if ctypes.get_errno() == ENOSYS and __name__ == "__main__":
```

Comment:
Nit: Since this is used so often a simple function
```python
def is_main() -> bool:
    return __name__ == "__main__"
```
or a global variable `IS_MAIN = __name__ == "__main__"`, would make this a bit nicer.